### PR TITLE
🧱 HOCS-5439: Ubuntu core based images

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,16 @@
-FROM quay.io/ukhomeofficedigital/hocs-base-image
+FROM quay.io/ukhomeofficedigital/hocs-base-image:4a4baa16a08fc6cbbf4369a2c922ae9caf43cc45
 
 USER 0
 
-RUN apk add --no-cache postgresql-client aws-cli ca-certificates curl bash
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y postgresql-client awscli ca-certificates  && \
```
Comment (zuzak) [inline Dockerfile]: Alphabetical order:
```suggestion
    apt-get install -y awscli ca-certificates postgresql-client && \
```

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,16 @@
-FROM quay.io/ukhomeofficedigital/hocs-base-image
+FROM quay.io/ukhomeofficedigital/hocs-base-image:4a4baa16a08fc6cbbf4369a2c922ae9caf43cc45
```
Comment (zuzak) [inline Dockerfile]: I'm not sure pegging to a specific version is a good idea: are you confident we'll remember to update this hash next time base-image has an update?

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
 
 USER 0
 
-RUN apk add --no-cache postgresql-client aws-cli ca-certificates curl bash
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y postgresql-client awscli ca-certificates  && \
+    rm -rf /var/lib/apt/lists/*
```
Comment (zuzak) [inline Dockerfile]: Consider
```suggestion
    apt-get clean
```

**Comment by zuzak** [comment]
Your commit message [should](https://chris.beams.io/posts/git-commit/) be an instruction:
> HOCS-5439: Use Ubuntu-based Docker base image

Moreover you should use the body of the commit message to explain why you are making the change.

File: Dockerfile (hunk 2 above)
Comment (nickmcmahon01) [reply Dockerfile]: Good spot, this is accidental

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**